gpgme-sharp/Context.cs

```diff
@@ -74,7 +65,7 @@ private void Initialize(ref ContextSigners signers, ref ContextSignatureNotation
             switch (errcode)
             {
                 case gpg_err_code_t.GPG_ERR_NO_ERROR:
-                    _ctx_ptr = ptr;
+                    CtxPtr = ptr;

```

Comment:
Where is declared CtxPtr? I don't see it.

Reply:
It's on line 94.

Reply:
Ok thanks. I have seen it now.